apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -133,6 +133,10 @@ def metadata
       ood_app.metadata.merge(form_config.fetch(:metadata, {}))
     end
 
+    def ssh_allow?
+      form_config[:ssh_allow]
+    end
```

Comment:
The default here is `nil` (`false`)?  Seems it should be `true`.

Reply:
Allowing this to return to `nil?` makes it easier to know whether we're falling back on the global configuration in the `ssh_to_compute_node?` method - I assume if `ssh_allow` isn't specified in the form, they want to fall back on the global option, not necessarily default to true.

Reply:
Nevermind on this question, we're using `nil` as a valid configuration. 

I.e., 
`true` - enable the button
`false` - disable the button
`nil` - no configuration, so check other configs.